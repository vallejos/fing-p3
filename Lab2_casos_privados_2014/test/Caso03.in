15
1 -1
3 -1
0 -1 
2 -1
-1
6 -1
5 -1
8 -1
9 -1
7 -1
11 13 -1
14 -1
10 4 -1
12 3 -1
10 13 7 -1
Benedetti 500 DelmiraAgustini 50 Ibarbouru 10 Rodo 1000 -1 -1
Benedetti 500 DelmiraAgustini 50 Ibarbouru 10 Rodo 1000 -1 -1
Benedetti 500 DelmiraAgustini 50 Ibarbouru 10 Rodo 1000 -1 -1
Benedetti 500 Ibarbouru 20 Rodo 1000 -1 -1
Onetti 1000 Ibarbouru 100 -1 -1
VazFerreira 1500 HerrerayReissig 150 DelmiraAgustini 50 -1 -1
VazFerreira 500 DelmiraAgustini 50 -1 -1
Quiroga 300 DelmiraAgustini 3 -1 -1
Quiroga 500 DelmiraAgustini 3 -1 -1
Quiroga 700 DelmiraAgustini 4 -1 -1
Galeano 300 HerrerayReissig 4 -1 -1
Galeano 400 HerrerayReissig 4 -1 -1
Galeano 500 HerrerayReissig 4 -1 -1
Galeano 800 HerrerayReissig 4 -1 -1
Galeano 200 HerrerayReissig 4 -1 -1
identificarColecciones 8 Benedetti Onetti Quiroga HerrerayReissig DelmiraAgustini Ibarbouru Galeano VazFerreira
identificarColecciones 9 Benedetti Onetti Quiroga HerrerayReissig DelmiraAgustini Ibarbouru Galeano VazFerreira Rodo
salir
