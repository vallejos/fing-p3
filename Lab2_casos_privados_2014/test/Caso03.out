Tarea 2 - 2014 - Programacion 3.

>0 -> Benedetti
1 -> Onetti
2 -> VazFerreira
3 -> Quiroga
4 -> Galeano
>0 -> Rodo
1 -> Onetti
2 -> VazFerreira
3 -> Quiroga
4 -> Galeano
>FIN.
